src/panel/direction/direction_panel.js

```diff
@@ -131,7 +132,7 @@ export default class DirectionPanel {
       if(routeParams.length > 0) {
         return `?${routeParams.join('&')}&vehicle=${this.vehicle}`
       } else {
-        return '?'
+        return true
       }
     } else {
       return ''
```

Comment:
This function now return either a boolean, or a string :thinking: 
It does not look right :suspect: